book.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * @wordpress-plugin
+ * Plugin Name:       Book List
+ * Description:       A plugin to manage a book list/post.
+ * Version:           0.1
+ * Author:            Ionut .T
+ */
+
+
+//hooks executed by my plugin
+add_action( 'wp_enqueue_scripts', 'book_register_scripts' );
+add_action( 'wp_enqueue_scripts', 'book_enqueue_scripts' );
+add_action( 'init', 'rest_api_init' );
+add_action( 'rest_api_init', 'register_rest_routes' );
+add_shortcode( 'book', 'load_short_code' );
+add_action( 'template_redirect', 'add_id' );
+
+/**
+ * Registers the scripts from the book plugin
+ * @return void
+ */
+function book_register_scripts() {
+	wp_register_script( 'backbone', includes_url('\js\backbone.js'), array( 'backbone' ) );
```

Comment:
you don't need to register backbone in order to use it. Just pass it as dependency for your scripts